### PR TITLE
[ENG-4340] Refactored all the bootstrap css out of the templates

### PR DESCRIPTION
**Pull request by bp-cos**
-   Ticket: [ENG-4340]
-   Feature flag: n/a

## Purpose

Refactored all the bootstrap css out of the templates

## Summary of Changes

Refactored all the bootstrap css out of the templates

## Screenshot(s)
Carousel - Desktop - Before
Carousel-Desktop-Before

Carousel - Desktop - After
Carousel-Desktop-After

Footer - Desktop - Before
Footer - Desktop - Before

Footer - Desktop - After
Footer - Desktop - Afterrousel-Desktop-Before.png…]()

Footer - Mobile - Before
Footer - Mobile - Before

Footer - Mobile - After
Footer Mobile - After

Join Banner - Desktop - Before
Join Banner - Desktop - Before

Join Banner - Desktop - After
Join Banner - Desktop - After

Join Banner - Mobile - Before
Join Banner - Mobile - Before

Join Banner - Mobile - After
Join Banner - Mobile - After

Password - Desktop - Before
Password - Desktop - Before

Password - Desktop - After
Password - Desktop - After

Password - Mobile - Before
Password - Mobile - Before

Password - Mobile - After
Password - Mobile - After

Sign-up - Desktop - Before
Sign-up - Desktop - Before

Sign-up - Desktop - Before
Sign-up - Desktop - After

Sign-up - Mobile - Before
Sign-up - Mobile - Before

Sign-up - Mobile - Before
Sign-up - Mobile - After

Registration Cards - Desktop
Registration Cards - Desktop

Registration Cards - Mobile
Registration Cards - Mobile

## Side Effects

Mass Chaos

## QA Notes

Check the attached images

[ENG-4340]: https://openscience.atlassian.net/browse/ENG-4340?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lib/osf-components/addon/components/institution-carousel/styles.scss [hunk 1]
```diff
@@ -14,17 +14,25 @@
             margin-top: -20px;
         }
     }
-}
 
-.InstitutionCarousel__item {
-    display: inline-block;
-    float: none;
+    .institution {
+        width: 16.66667%;
```
Comment (brianjgeiger) [inline lib/osf-components/addon/components/institution-carousel/styles.scss:19]: That's a very precise width percentage.

File: lib/osf-components/addon/components/node-card/template.hbs [hunk 2]
```diff
@@ -116,44 +116,54 @@
                     <dl>
                         {{#if @node.isRegistration}}
                             <div>
-                                <dt data-test-registration-template-label>
+                                <dt data-test-registration-template-label
+                                >
```
Comment (brianjgeiger) [inline lib/osf-components/addon/components/node-card/template.hbs:120]: Did you mean to split this (and some of the others below) into multiple lines?

**Review by brianjgeiger — COMMENTED** [review]
Very nice. Thank you for all the hard work on this. Hopefully the rest will be easier.

It looks like the node card went through a couple of noticeable changes; we might want to verify those with Mark. Also, could we get some screenshots of node cards for registrations (from the My Registrations page's submitted tab)?

**Review by futa-ikeda — CHANGES_REQUESTED** [review]
Found a couple leftover bootstrap classes from a cursory review. Will check this out locally and go over more thoroughly tomorrow.

**Review by brianjgeiger — APPROVED** [review]

**Pull request merged**